dex/src/main/scala/com/wavesplatform/dex/error/MatcherError.scala

```diff
@@ -595,6 +595,9 @@ case class SubscriptionsLimitReached(limit: Int, id: String)
 case class InvalidAddress(reason: String)
     extends MatcherError(address, commonEntity, commonClass, e"Provided address in not correct, reason: ${Symbol("reason") -> reason}")
 
+case class InvalidDepth(reason: String)
+  extends MatcherError(depth, commonEntity, commonClass, e"Provided depth in not correct, reason: ${Symbol("reason") -> reason}")
```

Comment:
Hm... I think, it is a part of request (as `RequestArgumentInvalid`). Consider to switch to `(request, depth, commonClass)`

Reply:
ok